client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -650,10 +650,27 @@ function isCamundaTab(tab) {
   return tab && [
     'bpmn',
     'cmmn',
-    'dmn'
+    'dmn',
+    'form'
   ].includes(tab.type);
 }
 
+function getDeploymentType(tab) {
```

Comment:
I added this function to decide the notification title based on the active tab type

Reply:
We still allow cmmn which is not covered at the moment.

Reply:
![image](https://user-images.githubusercontent.com/28307541/150107085-bf71b2da-3792-4631-928c-947103255dc1.png)

Reply:
So in that case it would be "Case definition deployed"

Reply:
I will also gladly approve a solution where "Process definition" is a fallback so that we don't have to handle CMMN specifically.

Reply:
Since it's only one more case to handle, I don't see an issue with adding it specifically. And it makes more sense in terms of UX